MetaMask/metamask-extension
# Update Bridge button visibility

## **Description**
* Updates bridge constants with new supported networks: BASE_MAINNET and LINEA_MAINNET. This enables the Bridge button in the EthOverview page when either network is selected
* Removes token allowlist from bridge constants. This makes the Bridge button visible in the TokenOverview page for any token, as long as the chain is allowlisted

## **Related issues**
N/A

## **Manual testing steps**

1. Open the extension
2. Select the Base network
3. Verify that Bridge button is enabled and opens the portfolio page when clicked
3. Switch to the Linea network
4. Verify that Bridge button is enabled and opens the portfolio page when clicked
6. Import the IZI token on Linea: 0x60d01ec2d5e98ac51c8b4cf84dfcce98d527c747
7. The Bridge button should be visible in the TokenOverview page
8. Clicking on the Bridge button opens the portfolio page with the IZI token selected

## **Screenshots/Recordings**

|Before| After |
|-|-|
| <img width="361" alt="Screenshot 2023-11-03 at 1 23 57 PM" src="https://github.com/MetaMask/metamask-extension/assets/100321200/6c6d72aa-3762-4f73-8201-527ca9f5ec94">| <img width="361" alt="Screenshot 2023-11-03 at 1 22 58 PM" src="https://github.com/MetaMask/metamask-extension/assets/100321200/12ba801c-3281-44ef-88c0-40f73c9ab2cf">|
| <img width="365" alt="Screenshot 2023-11-03 at 1 24 15 PM" src="https://github.com/MetaMask/metamask-extension/assets/100321200/60fc5a82-0371-4b44-922c-a87fe6d1107b">|<img width="360" alt="Screenshot 2023-11-03 at 1 27 33 PM" src="https://github.com/MetaMask/metamask-extension/assets/100321200/2ef380be-f066-4e66-99bf-c1562ec250b9"> |
| <img width="361" alt="Screenshot 2023-11-03 at 1 24 03 PM" src="https://github.com/MetaMask/metamask-extension/assets/100321200/21f4c06c-1c4f-4a7c-8388-ba91c774b1a5">| <img width="359" alt="Screenshot 2023-11-03 at 1 24 47 PM" src="https://github.com/MetaMask/metamask-extension/assets/100321200/30b6fcc8-294d-4809-a418-0dd2be4ddec3">|


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
